models/multimeter.cpp

```diff
@@ -103,8 +103,8 @@ nest::Multimeter::Parameters_::set( const DictionaryDatum& d,
   {
     throw BadProperty(
       "The recording interval, the interval offset and the list of properties "
-      "to record "
-      "cannot be changed after the multimeter has been connected to nodes." );
+      "to record cannot be changed after the multimeter has been connected "
```

Comment:
This needs a space at the end of the string, since the message output mechanism may change line breaks when the message is printed.